mods/ra/rules/structures.yaml

```diff
@@ -1719,7 +1725,7 @@ FIX:
 	Tooltip:
 		Name: Service Depot
 	Building:
-		Footprint: _X_ xxx _X_
+		Footprint: _=_ xxx _=_
```

Comment:
The top side of the SD looks to be perfectly left alone. The bottom is slightly run over but doesn't look intrusive as units are crossing a flat structure. IMO a good trade-off for mobility.